### DISCUSSION TITLE
New options for timeline component

### DISCUSSION THREAD

**Discussion by avido** [bladewindui/ui#172] [General]
I'd like to see an alignment  options for the timeline component. (For example: https://vuetifyjs.com/en/components/timelines/#alignment) 

The timeline component from vuetifyjs has more good options tough. But the alignment is very much wanted.

Cheers

**Comment by mkocansey** [comment]
@avido Just took a look at the vuetifyjs timeline components and agree there are a couple of options BladewindUI can "borrow" :). Will add them to the roadmap and keep you posted.

**Reply by mkocansey** [reply]
@avido The [timeline](https://bladewindui.com/component/timeline) got some redesign and a few more options

**Reply by avido** [reply]
yes yes yes !!! Perfect !
